neo/SmartContract/Manifest/ContractManifest.cs

```diff
@@ -41,6 +41,11 @@ public class ContractManifest : ISerializable
         /// </summary>
         public ContractAbi Abi { get; set; }
 
+        /// <summary>
+        /// Contract details including author and contact email.
+        /// </summary>
+        public ContractContactInformation Contact { get; set; }
```

Comment:
Metadata?

Reply:
It can be, I thought meta data was too abstract

Reply:
Contract remember me the contract it self :P

Reply:
What do you mean? it is written _contact_.
I think that other information can be added as "custom data". Like when you register a domain, you need a name and contact e-mails, in case  someone has to contact you.

Reply:
hahahahah

Reply:
Now, if I read well, is better :P